src/components/Form.js

```diff
@@ -40,10 +40,11 @@ const Form = () => {
     dispatch(postBook(newBook));
   };
   return (
-    <form onSubmit={(e) => submitBookToStore(e)} className="book-form" required>
-      <label htmlFor="book">
-        ADD NEW BOOK
-        <br />
+    <>
+      <hr />
+      <h1 className="form-title"> ADD NEW BOOK</h1>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you stick to the project styling as much as possible.
- For the `Fantasy` and `Action` part, you applied the bold property which was not applied to the main design
- For `The Hunger Games` and `Dune` Part, you should use the font `RobotoSlab` but you used `Montserrat`
- For the part where we have `Current Chapter` and `Chapter 17`, you applied the bold property. Kindly ensure your design looks more like the design template provided.
### Your Design
![Screenshot from 2022-04-02 14-21-26](https://user-images.githubusercontent.com/58692590/161385594-8060e3fc-ea15-47f1-b8c9-6e4261152fb6.png)
### Design Template
![Screenshot from 2022-04-02 14-21-18](https://user-images.githubusercontent.com/58692590/161385609-ce912eb6-bc64-4ee1-83a1-e5fd213733d2.png)